### PR TITLE
feature: [Text] Add isParagraph props

### PR DESCRIPTION
**Pull request by nicosampler**
This Pull Request adds a prop `isParagraph` to `Text` component.

When `isParagraph` (default true) Text is wrapped into a  `<p>` element.
When `isParagraph` is false, Text is Wrapped into a `<span>` element.

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr60--safereactcomponents.review.gnosisdev.com

File: src/dataDisplay/Text/index.tsx [hunk 1]
```diff
@@ -14,12 +14,27 @@ type Props = {
...
   className?: string;
+  is?: 'span' | 'paragraph';
 };
 
-const StyledText = styled.p<Props>`
+const SpanOrParagraph = React.forwardRef<
+  HTMLParagraphElement | HTMLSpanElement,
+  Props
+>(({ is = 'paragraph', children, ...rest }, ref) =>
+  is === 'paragraph' ? (
```
[the remark this answers is not in the document]
Comment (nicosampler) [reply src/dataDisplay/Text/index.tsx:24]: This produces a lot of problems with typescript, how would you pass the Props and also with the ref. 
how do you suggest to fix them?

[the remark this answers is not in the document]
Comment (mikheevm) [reply src/dataDisplay/Text/index.tsx:24]: https://styled-components.com/docs/api#as-polymorphic-prop

looks like styled-components has support for that. Could you please check it again and see if any errors pop up?

[the remark this answers is not in the document]
Comment (nicosampler) [reply src/dataDisplay/Text/index.tsx:24]: nice!

[the remark this answers is not in the document]
Comment (nicosampler) [reply src/dataDisplay/Text/index.tsx]: works like a charm!

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr60--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr60--safereactcomponents.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr60--safereactcomponents.review.gnosisdev.com

**Review by mikheevm — APPROVED** [review]

**Pull request merged**